edge-agent/test/Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test/ModuleManagementHttpClientTest.cs

```diff
@@ -21,7 +21,7 @@ namespace Microsoft.Azure.Devices.Edge.Agent.Edgelet.Test
     [Unit]
     public class ModuleManagementHttpClientTest : IClassFixture<EdgeletFixture>
     {
-        static readonly string[] Versions = new string[] { "2018-06-28", "2019-01-30", "2019-10-22" };
+        static readonly string[] Versions = new string[] { "2018-06-28", "2019-01-30", "2019-10-22", "2021-12-07" };
```

Comment:
can you please add all other missing versions (2020-07-07 and 2019 I see there are missing 2)

Reply:
db6c8f225bd5d93599035aead93a6a56e9326680